uiplib/scheduler.py

```diff
@@ -19,11 +19,25 @@ def __init__(self, offline, pics_folder, timout, website, count):
         self.timeout = timout
         self.directory = pics_folder
         self.count = count
+
         if not offline:
-            fetch = Thread(target=self.initFetch)
-            # all child threads need to be daemons to die upon main thread exit
-            fetch.setDaemon(True)
-            fetch.start()
+            try:
+                #Init the threads
+                unsplashThread = Online_Fetch(1,self.website[0],self.directory,
```

Comment:
@abhsag24 I had thought on it... But creating a thread for calling another function that itself is a threaded function is not suitable I guess. And threading here is directly is better, as it doesnt wait for the 1st url to scrape images completely, instead downloades image from both the websites concurrently.